deltakit-core/src/deltakit_core/decoding_graphs/_explained_dem_parsing.py

```diff
@@ -41,7 +41,7 @@ def depolarising_as_independent(probability: float, num_qubits: int) -> float:
         )
     p_with_i = probability / mixing_probability
     exponent = 1 / 2 ** (2 * num_qubits - 1)
-    return (1 - ((1 - p_with_i) ** exponent)) / 2
+    return float((1 - (1 - p_with_i) ** exponent) / 2)
```

Comment:
I looked into this. I guess `mypy` doesn't have very good inference when there are `**`. I don't think there's much to be done except cast the result.